src/Analyzers/Core/CodeFixes/PopulateSwitch/AbstractPopulateSwitchCodeFixProvider.cs

```diff
@@ -58,8 +58,8 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
         {
             var diagnostic = context.Diagnostics.First();
             var properties = diagnostic.Properties;
-            var missingCases = bool.Parse(properties[PopulateSwitchStatementHelpers.MissingCases]);
-            var missingDefaultCase = bool.Parse(properties[PopulateSwitchStatementHelpers.MissingDefaultCase]);
+            var missingCases = bool.Parse(properties[PopulateSwitchStatementHelpers.MissingCases] ?? string.Empty);
```

Comment:
>string.Empty [](start = 101, length = 12)

`bool.Parse()` will throw an exception parsing `string.Empty` (or `null`). Presumably we're not passing those values now so perhaps we should use `bool.Parse(properties[...]!)` instead.

Reply:
True, `""` will throw, just like `null` would (but different exceptions, and with this change we don't get a build warning). 

Sure, I could use `!` but there's in fact no guarantee that the value *isn't* null as far as I could tell. I suppose if the code fix provider wants to make an assumption about the analyzer never setting a null value, that's fine. I'll change to the `!` here.